app/forms/previousRegistrations/PreviousEuVatNumberFormProvider.scala

```diff
@@ -30,6 +30,6 @@ class PreviousEuVatNumberFormProvider @Inject() extends Mappings {
       "value" -> text("previousEuVatNumber.error.required", Seq(country.name))
         .verifying(firstError(
           maxLength(12, "previousEuVatNumber.error.length"),
-            regexp(euVatNumberPattern, "previousEuVatNumber.error.invalid")))
+            regexp(euVatNumberPattern, "previousEuVatNumber.error.invalid"))).transform[String](_.toUpperCase, x => x)
```

Comment:
Could we rename x to make it something more descriptive of what it actually is, and may aswell break it onto a new line as it is the same level as the .verifying